python/tests/test_search.py

```diff
@@ -0,0 +1,46 @@
+from sidecar import search
+
+def test_search(monkeypatch):
```

Comment:
```suggestion
def test_search(monkeypatch, capsys):
```

Reply:
resolved in https://github.com/refstudio/refstudio/pull/250/commits/b716691359753cba88ef5d0ef28f3b73e494fe8c